department-of-veterans-affairs/vets-website
# Pension 70423 mvp home acreage

## Summary
On the Pension Benefits application(21P-527EZ), update the 'Financial information' chapter to collect more information

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Start or continue the application
3. Go to the 'Financial Information' chapter
4. Continue to the 'Income and assets' home ownership page
5. Verify that the 'Income and assets' home acreage page is the next page when 'Yes' is selected
6. Verify that the 'Income and assets' home acreage page is NOT the next page when 'No' is selected
10. Verify the layout and content
11. Verify that the form data is saved to the redux store and submitted to the api

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70423)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots

### Desktop
| Home Ownership Acreage | 
| ------ |
|![localhost_3001_pension_application_527EZ_ (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/edaf23f7-e64f-417f-9879-5683b7d14361)

## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70423)

### Quality Assurance & Testing

### Error Handling


### Authentication
